.github/workflows/E2E.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     name: Test on Linux with Node ${{ matrix.node }}
     strategy:
       matrix:
-        node: [14, 16, 18]
+        node: [18]
```

Comment:
Node v16 is still LTS. I recommend following [the same node.js support policy that Hardhat uses](https://hardhat.org/hardhat-runner/docs/reference/stability-guarantees#node.js-versions-support)  and documenting it somewhere.

Reply:
Good note, I opted to leave v19 out since it's got a week of support left. Initially I removed them because I didn't want to run out of github runners' free tier, but I guess it's better to cross that bridge when I get to it.